source/revanc.c

```diff
@@ -101,7 +101,7 @@ int brute_force_evict_set(struct page_format *fmt, void *evict_target,
 				slot &= level->slot_mask;
 				expected_slot &= level->slot_mask;
 
-				if (fabs(slot - expected_slot) <= 1.0) {
```

Comment:
I recall casting these values to a float in an earlier version when I was testing this, but I might have accidentally omitted the type cast. Removing the `fabs()` is definitely not correct.